scripts/seedDatabase.ts

```diff
@@ -0,0 +1,55 @@
+import { initializeApp } from "firebase/app";
+import { firebaseConfig } from '../src/firebase'
+import { getFirestore, setDoc, doc } from "firebase/firestore";
+import { channels } from '../src/config/ambiance/channels';
+import { ambianceCategoryDetail, ambianceCategories } from "../src/config/ambiance/index";
+
+
+const app = initializeApp(firebaseConfig);
+const db = getFirestore(app);
+
+//addChannels();
+//addWarcraft();
```

Comment:
So since these are both commented out, if we ran this script nothing would happen but if I didnt know about this script I could assume that it might have properly ran the seed. Can we make it so we log what is seeded and enable these. For example:
```
Seeding Channels...

...

Channels Seeded!
```

Reply:
Could also add some sort of script params to do this:
```
yarn seedDatabase channels
yarn seedDatabase ambiance
```